src/test/python/MCSVerification.py

```diff
@@ -0,0 +1,640 @@
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import getopt
+import glob
+import json
+import numpy
+import os
+import subprocess
+import sys
+import urllib
+import zipfile
+from collections import Counter
+from datetime import datetime, timedelta
+from itertools import groupby
+from netCDF4 import Dataset
+
+def _get_python_implementation():
+    '''
+    Purpose: To get the results of python implementation from the SciSpark website
+    Inputs: None
+    Outputs: Boolean representing success retrieving the data
+    '''
+    print 'Checking on Python implementation data' 
+    of.write('\n Checking on Python implementation data')
+
+    pythonGTG = 'https://scispark.jpl.nasa.gov/pythonGTG.zip'
+
+    os.chdir(workingDir)
+    try:
+        if not os.path.exists(workingDir + '/pythonGTG'):
+            urllib.urlretrieve(pythonGTG, workingDir + '/pythonGTG.zip')
+            zipGTG = zipfile.ZipFile(workingDir + '/pythonGTG.zip', 'r')
+            zipGTG.extractall(workingDir)
+            zipGTG.close()
+            os.remove(workingDir + '/pythonGTG.zip')
+
+        print 'Python implementation data retrieved from %s' %pythonGTG
+        of.write('Python implementation data retrieved from %s' %pythonGTG)
+        return True
+    except:
+        print '!!Problem retrieving %s' %pythonGTG
+        of.write('\n!!Problem retrieving %s' %pythonGTG)
+        return False
+
+def _run_scispark_implementation():
+    '''
+    Purpose: To run GTG runner for SciSpark results to compare as a spark-submit task
+    Assumptions: spark-submit is available on the path. scala-2.11 is used.
+    '''
+    runSpark = False
+
+    print 'Checking on SciSpark implementation data' 
+    of.write('\n Checking on SciSpark implementation data')
+
+    try:
+        if not os.path.exists(workingDir + '/scisparkGTG'):
+            os.mkdir(workingDir + '/scisparkGTG')
+            runSpark = True
+        if not os.path.exists(workingDir + '/scisparkGTG/textFiles'):
+            os.mkdir(workingDir + '/scisparkGTG/textFiles')
+            runSpark = True
+        if not os.path.exists(workingDir + '/scisparkGTG/MERGnetcdfCEs'):
+            os.mkdir(workingDir + '/scisparkGTG/MERGnetcdfCEs')
+            runSpark = True
+
+        if runSpark:
+            os.chdir(workingDir + '/../../../../')
+
+            sparkSubmitStr = 'spark-submit target/scala-2.10/SciSpark.jar'
```

Comment:
@kwhitehall 
Do we want to use spark-submit for the verification.
For the purposes of the test, you are using the resources/Papersize directory which doesn't require an entire cluster to run. Would it be better to just run "sbt run". Otherwise, to run the script users will need to have installed spark on their local machine.